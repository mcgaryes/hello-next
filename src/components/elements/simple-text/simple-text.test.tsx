import {render} from '@testing-library/react'
import SimpleText from "@/elements/simple-text/simple-text";

test('Renders simple-text unchanged', () => {
    const {container} = render(<SimpleText />)
    expect(container).toMatchSnapshot()
})
