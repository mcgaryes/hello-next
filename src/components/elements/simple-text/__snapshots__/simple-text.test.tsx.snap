// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders simple-text unchanged 1`] = `
<div>
  <div />
</div>
`;
